TanukiKitTests/ProjectTests.swift

```diff
@@ -6,7 +6,7 @@ class ProjectTests: XCTestCase {
     // MARK: Actual Request tests
 
     func testGetProjectsCOM() {
-        let session = TanukiKitURLTestSession(expectedURL: "https://gitlab.com/api/v3/projects?access_token=12345&page=1&per_page=100", expectedHTTPMethod: "GET", jsonFile: "Projects", statusCode: 200)
+        let session = TanukiKitURLTestSession(expectedURL: "https://gitlab.com/api/v3/projects?access_token=12345&archived=&order_by=created_at&page=1&per_page=20&search=&simple=&sort=desc&visibility=", expectedHTTPMethod: "GET", jsonFile: "Projects", statusCode: 200)
```

Comment:
`&search=&simple=&sort=desc&visibility=`

This looks very weird. If the parameter name is not given it should not be added to the request. Empty parameter names can leas to unexpected behavior.